### PR TITLE
Tickets > Quantity markup

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/ETP-224
🎥 https://www.loom.com/share/5c87563c7a8b4966a847c6859e24694d

### CODE REVIEW

File: src/views/blocks/tickets/quantity-remove.php [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
- * @version 4.11.4
+ * @version TBD
  *
  * @var $this Tribe__Tickets__Editor__Template
  */
 
 $ticket = $this->get( 'ticket' );
 $button_title = sprintf(
-	_x('Decrease ticket quantity for %s', '%s: ticket name.', 'event-tickets'),
+	_x( 'Decrease ticket quantity for %s', '%s: ticket name.', 'event-tickets' ),
```
Comment (tr1b0t) [inline src/views/blocks/tickets/quantity-remove.php]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8eb0acd7a3a98a976681aa50c87fca9b</sub>

Comment (sc0ttkclark) [reply src/views/blocks/tickets/quantity-remove.php]:
```suggestion
	/// translators: %s: ticket name.
	_x( 'Decrease ticket quantity for %s', 'Decrease ticket quantity button title', 'event-tickets' ),
```

File: src/resources/postcss/tickets.pcss [hunk 2]
```diff
@@ -303,10 +303,6 @@
 		}
 	}
 
-	.tribe-tickets__item__quantity__add {
```
Comment (sc0ttkclark) [inline src/resources/postcss/tickets.pcss]: Was removing this intentional? Didn't see a changelog note about it

Comment (juanfra) [reply src/resources/postcss/tickets.pcss]: Yes -> https://github.com/moderntribe/event-tickets/pull/1628/commits/02a70ed829f98da5a7dd780ad947a31281452c51

File: readme.txt [hunk 3]
```diff
@@ -122,6 +122,8 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Update notice template to print content only when there's information. Remove the "Whoops" title for the Attendee Registration page notice. [ETP-233]
 * Fix - Update file path in the docblocks of the templates for The Events Calendar new views. [ETP-289]
 * Fix - Update position of the `tribe_tickets_ticket_add` action so it receives the ticket data on ticket creation. [ETP-302]
+* Fix - Removed the `type="submit"` from button element in the ticket quantity remove template, as it's its default. [ETP-224]
```
Comment (sc0ttkclark) [inline readme.txt:125]:
```suggestion
* Fix - Removed the `type="submit"` from the button element in the ticket quantity remove template. [ETP-224]
```

File: readme.txt [hunk 4]
```diff
@@ -122,6 +122,8 @@ Currently, the following add-ons are available for Event Tickets:
...
+* Fix - Removed the duplicate `type="button"` from button element in the ticket quantity add template. [ETP-224]
```
Comment (sc0ttkclark) [inline readme.txt:126]:
```suggestion
* Fix - Removed the duplicate `type="button"` from the button element in the ticket quantity add template. [ETP-224]
```

File: src/views/blocks/tickets/quantity-add.php [hunk 5]
```diff
@@ -13,20 +13,21 @@
...
  * @since 4.11.4 Added accessibility classes to screen reader text element.
+ * @since TBD    Removed duplicate `type="button"` from button element.
  *
- * @version 4.11.4
+ * @version TBD
  *
  * @var $this Tribe__Tickets__Editor__Template
  */
 
 $ticket = $this->get( 'ticket' );
```
Comment (tr1b0t) [inline src/views/blocks/tickets/quantity-add.php:23]: :warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space

<sub>hash: 454b45d1160d913234476ad142b23c7b</sub>

File: src/views/blocks/tickets/quantity-remove.php [hunk 6]
```diff
@@ -13,20 +13,21 @@
...
  * @since 4.11.4 Added accessibility classes to screen reader text element.
+ * @since TBD    Removed `type="submit"` from button element, as it's its default.
  *
- * @version 4.11.4
+ * @version TBD
  *
  * @var $this Tribe__Tickets__Editor__Template
  */
 
 $ticket = $this->get( 'ticket' );
```
Comment (tr1b0t) [inline src/views/blocks/tickets/quantity-remove.php:23]: :warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space

<sub>hash: a7caa4f7c67a08f8803239d194bec2ce</sub>

**Pull request merged**